awx/main/models/events.py

```diff
@@ -546,6 +549,13 @@ def _update_host_summary_from_stats(self, hostnames):
 
             Host.objects.bulk_update(list(updated_hosts), ['last_job_id', 'last_job_host_summary_id'], batch_size=100)
 
+            # bulk-create
+            current_time = now()
```

Comment:
@wenottingham @ryanpetrello  so just to verify, we're using time `now()` here, which will lead to slightly different numbers than in analytics, is that ok?

In analytics, we're currently exposing `job_created` timestamp, then timestamp of first/last finished task on host and then timestamp of first/last task with changed=True on the host.

Reply:
@Ladas this version of `now()` is timezone aware, not naive.  I would think it should be fine, unless analytics itself is not using aware timestamps.

Reply:
... unless I've misunderstood the objection, which upon second reading I think I may have.

Reply:
@jbradberry right, yeah I am basically saying this list will produce different timestamps than when you'll fetch this list from AA. (so trying to confirm, if the listing of the hosts like this from Tower and AA should produce identical lists or not :-) )

In here we're taking `now()` somewhere around processing of playbook on stats, right?

In AA we have multiple timestamps, but always taking the existing `created` timestamps of job/events, so not creating another timestamp.

Reply:
I'm interested in how much @wenottingham cares about this matching exactly.  My understanding is that this feature solely exists as a reporting mechanism for folks who *don't* use AA and want an idea of total automation count (unique hosts) over a date range.

Reply:
If we wanted to be *super* particular or improve this slightly, _maybe_ instead of using `now()` we could just use `self.created` (given that `self` is a `playbook_on_stats` `JobEvent`).

Reply:
Ideally, they would match. 

In the long run, it probably doesn't matter too much whether or not they do, as the difference over the course of the job isn't that meaningful. This won't catch playbooks that abort before the end, and the AA approach will, as I understand it, which is another differene.

Reply:
@ryanpetrello `created` timestamp of the `playbook_on_stats` event should be close enough to the `max(created)` of host tasks we're storing, so that should be ok I think. (we're storing timestamp of last task per host, so it might differ more for hosts processed early and later)